charts/radar-gateway/templates/deployment.yaml

```diff
@@ -69,6 +69,20 @@ spec:
           env:
           - name: JAVA_OPTS
             value: {{ include "radar-gateway.javaOpts" . }}
+          {{- if and .Values.secret.producer.jaas.name }}
+          - name: KAFKA_PRODUCER_SASL_JAAS_CONFIG
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.secret.producer.jaas.name }}
+                key: {{ .Values.secret.producer.jaas.key }}
```

Comment:
In other charts, e.g. catalog-server this is only set this way, if `cc.enabled` is false. Should there be a similar condition here? For radar-gateway also there are some properties set in the configmap, if cc.enabled. Should there be also some alternative in configmap for those settings for strimzi?

Reply:
Ah this is a nouanced su